components/autocomplete/events.md

```diff
@@ -227,6 +228,62 @@ When using `OnRead`, make sure to set `TItem` and `TValue`.
 }
 ````
 
+## OnItemRender
+
+The `OnItemRender` event fires when each item in the collection passed to the `Data` parameter of the AutoComplete renders. 
+
+The event handler receives as an argument an `AutoCompleteItemRenderEventArgs<TItem>` object that contains:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Property | Description                              |
+|----------|------------------------------------------|
+| `Item`   | The current item that renders in the AutoComplete. |
+| `Class`  | The custom CSS class that will be added to the item.     |
+
+````CSHTML
+@* Disable an item in the AutoComplete *@
```

Comment:
Consider custom styling based on a model value, rather than "disabled" item. It's not truly disabled, because you can select it via keyboard navigation. This will generate support or negative docs votes.

Reply:
We are implementing this to respond to the Disabled Items support in Selects (https://feedback.telerik.com/blazor/1474264-add-support-for-disabled-items).

Reply:
To add to this, maybe we can enhance the CSS rules and in the public feedback item showcase OnitemRender + cancel the OnClose event so it truly disables an item.

Reply:
You can change the selected item with the keyboard even without opening the dropdown. So `OnItemRender` is irrelevant here and the requested feature still does not exist.

Reply:
See ticket 1576049 that shows "more" "disabled" items. This can be a KB, while the `OnItemRender` example in the docs can focus on styling customizations to keep it simple, plus optional link to the KB.